Configurations/release-move-tag.sh

```diff
@@ -4,23 +4,30 @@ set -e
 # Convenience script to automatically commit Package.swift after updating the checksum and move the latest tag
 latest_git_tag=$(git describe --abbrev=0) # gets the latest tag name
```

Comment:
I just realized this isn't scanning lightweight tags. Can you add `--tags` to this `git describe` invocation? (other scripts already do this).

Reply:
Done in 363c05a1a87545fa06e42d0cb0892de7662ab8fd.